system-container/README.md

```diff
@@ -0,0 +1,60 @@
+# System Container support
+
+A system container runs using runC instead of Docker and it is managed by the
```

Comment:
Maybe better to say the process is managed by systemd and installed using the atomic tool.